### DISCUSSION TITLE
Make Freeplane's Tutorial and Documentation files correspond to online documentation?

### DISCUSSION THREAD

**Discussion by 1-DimensionalBox** [freeplane/docs#51] [General]
Hi all,

Reading through the 'First steps' and 'Formatting maps and nodes' pages, a thought struck me: wouldn't it be much easier to learn Freeplane from within the application?

I know that mind maps for learning already exist within Freeplane, which can be accessed through `Help/Tutorial` and `Help/Documentation`. Could it be a good idea to try to make these files correspond to our online documentation (and the other way around)?

**Comment by macmarrum** [comment]
Related discussions

* https://github.com/freeplane/docs/discussions/15
* SF [new ADDON Tutorial-o-Matic _#f50ef44f98_](https://sourceforge.net/p/freeplane/discussion/758437/thread/f50ef44f98/)
* SF [Migrating from SourceForge [Community Conference] (2) _#6fecc713d2_](https://sourceforge.net/p/freeplane/discussion/758437/thread/6fecc713d2/)
* https://github.com/freeplane/freeplane/discussions/261

**Reply by 1-DimensionalBox** [reply]
Thank you! Especially the Tutorial-o-Matic from @EdoFro seems relevant.

The other discussions are a bit more general on one side; on the other side I think it would be good to first generally discuss the topic before making an issue or pull request. That's why I started this discussion :slightly_smiling_face:
